types/cloudfour__simple-svg-placeholder/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for @cloudfour/simple-svg-placeholder 1.0
```

Comment:
It looks like the library was updated to support esm imports in v1.1. can you update this comment line to reflect that please?

https://github.com/cloudfour/simple-svg-placeholder/commit/4b9307b400b9957b5e43f6b55f260263fc150474

Reply:
Thanks for the swift response! I have updated the version accordingly